ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -3145,6 +3145,18 @@ HWZTC:
   implements:
   - CONTROL
 
+HWZTVC:
```

Comment:
Please update the abstract name to HWBZC

Reply:
@ccquigley - abstract name and entity type updated, thank you